cmd/bucket-handlers.go

```diff
@@ -618,6 +618,15 @@ func (api objectAPIHandlers) PutBucketHandler(w http.ResponseWriter, r *http.Req
 		return
 	}
 
+	// Make a DNS entry in Kubernetes Operator
```

Comment:
Same as below, the bucket creation on the operator is idempotent, so it is easier to create service then create the bucket and there is no need to clean up.